main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -327,9 +327,9 @@ class Bootstrap(val projectPath: Path, apiKey: Option[String]) {
   /**
     * Builds (compiles) the source files for the project.
     */
-  def build(flix: Flix): Validation[CompilationResult, BootstrapError] = {
+  def build(flix: Flix, build: Build = Build.Development): Validation[CompilationResult, BootstrapError] = {
```

Comment:
This is really just a first attempt. As you rework the package manager, you will have to keep track of this more precisely.

But I think it should be fairly straight forward.